cluster.py

```diff
@@ -8,14 +10,33 @@
 logger = logging.getLogger(__name__)
 
 
+def get_cluster_lock(timeout: int = 10 * 60) -> socket.socket:
+    logger.info("Acquiring cluster lock to prevent concurrent run on the same machine...")
+    sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    start_time = time.time()
+    while start_time + timeout > time.time():
+        try:
+            sock.bind(('127.0.0.1', 48783))  # random port
```

Comment:
Why just one single port? What about using pool of it?

SCT has following thing: https://github.com/scylladb/scylla-cluster-tests/blob/c2909803/sdcm/utils/common.py#L1772

Reply:
I just wanted to do quick solution.
I can add a pool and enable tests run in parallel.